src/tcache.c

```diff
@@ -756,21 +857,29 @@ tcache_create_explicit(tsd_t *tsd) {
 	 * the beginning of the whole allocation (for freeing).  The makes sure
 	 * the cache bins have the requested alignment.
 	 */
-	size_t size = tcache_bin_alloc_size + sizeof(tcache_t)
+	unsigned thread_nhbins = thread_nhbins_get(tsd);
```

Comment:
This is for the manual tcache case. Let's use the default settings here, instead of the ones from the auto tcache.